database/db.go

```diff
@@ -229,11 +242,18 @@ func NewDbFromConfig(c *Config, logger *logging.Logger, connectorCallbacks Retry
 	}, nil
 }
 
-// GetAddr returns the database host:port or Unix socket address.
+// GetAddr returns a URI-like database connection string of the following syntax: type[+tls]://user@host[:port]/database
 func (db *DB) GetAddr() string {
 	return db.addr
 }
 
+// MarshalLogObject implements zapcore.ObjectMarshaler, adding the database address, DB.GetAddr, to each log message.
```

Comment:
Even if this function is only used indirectly, I would add square brackets to `DB.GettAttr`, i.e. `[DB.GetAddr]` so that it is transformed to a [link](https://tip.golang.org/doc/comment#links) in the documentation. Same for Redis.